docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -277,6 +283,80 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending Search Requests from Client to Server
+
+Client application can send search requests, which consist of a collection of search conditions.
+Server listens for the incoming search requests and handles them accordingly.
+After handling a request, server sends a response to the client that sent the request.
+
+### Receiving Search Request
+
+To receive and handle search request on the server, follow these steps:
+
+1. Get server handle.
+
+```
+var server = tizen.mediacontroller.createServer();
+```
+
+2. Define search request callback:
+
+```
+function searchRequestCallback(client, request) {
+    console.log(client);
+    console.log(request);
+
+    // You can return RequestReply object, which will be sent to the client
+    // who can handle it in search request reply callback.
+    return new tizen.mediacontroller.RequestReply(123, {"someData": "someValue"});
+}
+```
+
+3. Register search request listener
```

Comment:
Please add a colon at the end of the sentence.

Register search request listener: